src/Common/Nav.elm

```diff
@@ -0,0 +1,25 @@
+module Common.Nav exposing (navBar)
```

Comment:
@andela-pwanjiru @akabiru Any particular reason we are reintroducing this file? 🤔 
I think this functionality is taken care of in the [app common nav](https://github.com/AndelaOSP/shufflebox-frontend/blob/develop/src/App/Common/Nav.elm).